assignments/objects.js

```diff
@@ -33,13 +63,30 @@ const example = {
 
 // Antonietta's Gender
 
+console.log(intern1.name);
+console.log(intern2.id);
+console.log(intern3.email);
+console.log(intern4.name);
+console.log(intern5.gender);
+
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+intern2.speak = function () {
+  console.log("Hello, my name is Kennan!");
+}
+console.log(intern2.speak());
+
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
 //console.log(antonietta.multiplyNums(3,4));
 
+intern5.multiply = function (a, b) {
+  let product = a * b;
+  console.log(product);
+}
+console.log(intern5.multiply(3,4))
```

Comment:
Same as the intern2.speak method, this is actually logging out undefined after it logs out the `console.log()` inside of the `intern5.multiply` method. That is because you are logging the return value of the `intern5.multiply` method, but you aren't returning anything in that function. 

How I would have done it:
```
intern5.multiply = function (a, b) {
  let product = a * b;
  return product;
}
```